src/domain/event/EventOccurrenceRedirect.tsx

```diff
@@ -16,6 +16,7 @@ import LoadingSpinner from '../../common/components/spinner/LoadingSpinner';
 import InfoPageLayout from '../app/layout/InfoPageLayout';
 import useEventOccurrence from './queries/useEventOccurrence';
 import useAriaLive from '../../common/AriaLive/useAriaLive';
+import useGetPathname from '../../common/route/utils/useGetPathname';
 
 function getEventPathname(pathname: string): string {
```

Comment:
Unused function

Reply:
The `useLocation` hook is also no longer used.

For some reason GH doesn't allow me to comment on that row directly.